pkg/vscode/v1/Makefile

```diff
@@ -0,0 +1,37 @@
+# GENERATE_VSCODE_SETTINGS can be set to false so that the user can opt
+# out of generating the VSCode settings file.
+GENERATE_VSCODE_SETTINGS ?= true
+
+# VSCODE_SETTINGS_FILE is the VSCode workspace settings file that contains
+# workspace-specific settings that override user-defined settings and their
+# defaults.
+# More about VSCODE workspace settings can be found at:
+# https://code.visualstudio.com/docs/getstarted/settings#_workspace-settings
+VSCODE_SETTINGS_FILE = $(MF_PROJECT_ROOT)/.vscode/settings.json
+
+ifeq ($(GENERATE_VSCODE_SETTINGS),true)
+GENERATED_FILES += $(VSCODE_SETTINGS_FILE)
+endif
+
+# VSCODE_SETTINGS_REQ is a space separated list of prerequisites requred to
+# generate the VSCode workspace settings file.
+VSCODE_SETTINGS_REQ ?=
+
+ifneq ($(filter proto,$(MF_LANGUAGES)),)
+VSCODE_SETTINGS_REQ += $(MF_PROJECT_ROOT)/artifacts/protobuf/args/common
+
+ifneq ($(filter go,$(MF_LANGUAGES)),)
+VSCODE_SETTINGS_REQ += $(MF_PROJECT_ROOT)/artifacts/protobuf/args/go
+endif
+
+endif
+
+RUN_PROTOC_SCRIPT = $(MF_PROJECT_ROOT)/artifacts/protobuf/bin/run-protoc
```

Comment:
I think it would be better if this variable (or something similar) were defined by the `proto` package, since it "owns" the script.

Also, it might be unimportant from the perspective of VSCode that it's a script, and not the real `protoc` binary, so maybe we could just call it `PROTO_PROTOC_PATH` or something like that, what do you think?